src/main/java/com/microsoft/azure/functions/worker/broker/JavaFunctionBroker.java

```diff
@@ -132,21 +135,53 @@ void registerWithClassLoaderProvider(File libDirectory) {
 
 				if(Helper.isLoadAppLibsFirst()) {
 					// load client app jars first.
-					classLoaderProvider.addDirectory(libDirectory);
-					classLoaderProvider.addDirectory(workerLib);
+					addDirectory(libDirectory, false);
```

Comment:
Java8:
1. load cx first with annotation lib  ---> cx annotation lib will be put in cxset, worker annotation lib will be put in workerset, order is expected
2. load cx first without annotation lib --> no cx annotation lib in cxset, worker annotation lib will be put in workerset, as expected
3. load worker first with annotation lib --> cx annotation lib in cxset, worker annotation lib in workerset, order as expected
4. load worker first without annotation lib --> no cx annotation lib in cxset, worker annotation lib will be put in workerset, as expected

Java11: should be same as v4